classes/class-uagb-coming-soon.php

```diff
@@ -43,7 +43,7 @@ public function __construct() {
 		$enabled_coming_soon = UAGB_Admin_Helper::get_admin_settings_option( 'uag_enable_coming_soon_mode', 'disabled' );
 		$coming_soon_page_id = UAGB_Admin_Helper::get_admin_settings_option( 'uag_coming_soon_page', false );
 
-		if ( 'enabled' === $enabled_coming_soon && ! is_user_logged_in() && false !== $coming_soon_page_id && isset( $coming_soon_page_id ) && ! empty( $coming_soon_page_id ) ) {
+		if ( ( 'enabled' === $enabled_coming_soon || 'maintenance' === $enabled_coming_soon ) && ! is_user_logged_in() && false !== $coming_soon_page_id && isset( $coming_soon_page_id ) && ! empty( $coming_soon_page_id ) ) {
```

Comment:
Will this work as intended?

```suggestion
		if ( 'disabled' !== $enabled_coming_soon && ! is_user_logged_in() && false !== $coming_soon_page_id && isset( $coming_soon_page_id ) && ! empty( $coming_soon_page_id ) ) {
```

Reply:
Works as intended, updated.